### ISSUE TITLE
Train-validation split

### ISSUE THREAD

**Issue by hosseinshn** [biomedia-mira/masf#11]
In the paper you mentioned that for VLCS you had 70-30 for train-val splits. Where in the code you defined this split? 
Similarly, for PACS you did not mention anything about validation set, because leave-one-out cross validation is leaving one domain entirely for independent test set. Does this mean you did not use any validation set for PACS?

**Comment by carrenD** [comment]
The random split of VLCS is conducted separately to generate a file list, for the ease of implementation. Yes, PACS entirely leaves one domain as test set, following the practice in literature.

**Comment by hosseinshn** [comment]
Thank you. Can you please provide the filelist that you used for VLCS?
